### DISCUSSION TITLE
Newbie issue: unable to successfully load any performances other than default

### DISCUSSION THREAD

**Discussion by bjchapm** [probonopd/MiniDexed#710] [Q&A] [answered]
Hello and sorry for such a basic question.

I've installed minidexed on a RPI 4B with a non-I2C display. For now, I'm using buttons to navigate the menus, using this config:

```
# GPIO Button Navigation
#  Any buttons set to 0 will be ignored
ButtonPinPrev=27
ButtonActionPrev=click
ButtonPinNext=18
ButtonActionNext=click
ButtonPinBack=7
ButtonActionBack=click
ButtonPinSelect=11
ButtonActionSelect=click
ButtonPinHome=11
ButtonActionHome=doubleclick
ButtonPinShortcut=11
# (Shortcut doesn't have an action)

```
Navigation and selection of items works. I'm using an Arturia Keystep as the controller, connected via USB.

When I boot, the default performance.ini appears to be loaded correctly and my keyboard correctly connects to the synth and generates notes using a slightly spacey synth lead sound, which I assume is coming from the default. Everything appears to work correctly.

When I navigate to the "Performance" menu and load any of the other performances, I get no sound at all. After attempting to load the `Concert D` performance, I see the following on the display, which makes me think that I've successfully loaded the performance:

```
 001:002 [L] Load
<Concert D          >
```

I've tried loading several performances but always get silence. As a further test, I tried saving this performance as the default and then rebooting. I confirmed that the performance.ini file is in fact overwritten. However, after rebooting, I still get silence. I also ran the `getsysex.sh` script and it correctly populated the `sysex/voice` folder, but that didn't resolve the issue.

I've searched through the discussions and the wiki and have not found a solution. 

What am I doing wrong?

Thank you for this fantastic project and for any help you can give,

Ben

**Comment by Banana71** [comment, marked as answer]
Hi Ben, 

Except for the default Performance, all Performances only receive on MIDI channel 1. The default Performance on a fresh system receives on all TGs in Omni mode.
You probably just need to set the MIDI channel on your KeyStep to 1.


Peter
![KeyStep CH1](https://github.com/user-attachments/assets/c4afe077-bcc1-4024-b0a6-8db13d581c52)

**Reply by bjchapm** [reply]
Peter,

Thank you so much! That was exactly the issue. Now to explore all of the performances! 

Best wishes, Ben

**Reply by lgtech6719** [reply]
Is it possible to use performance on omni channel ? I am using a guitar midi controller (fender mustang Xbox controller). Each string will function in different midi channel ie, 1-6 channels. when using with TG1 to TG8 , all strings works. But with Performance , only 1st string will work ie, midi channel 1 only.

**Reply by Banana71** [reply]
Yes, it is possible for a performance to receive MIDI data on all channels. Unfortunately, the miniDexed does not yet have the option to select the receiving channel globally. You have to set the MIDI channel to > 16 for each TG in each performance you want to play. You can then save the performance.

```
MIDIChannel#=17          # 1 .. 16, 0: off, >16: omni mode
```

**Reply by lgtech6719** [reply]
Thank you for the response. So I have to change each performance ini file midichannel to omni manually, that's right?

**Reply by Banana71** [reply]
Yes, unfortunately that is still the case. You can adjust the settings directly on the miniDexed or you can do it with the help of a PC.

**Reply by lgtech6719** [reply]
Thank you
